PayButton/paybutton-server
# [Issue #4] feat: create transactions page

**Description:**
Creates a primitive front-end for transactions. Right now this only works for BCH addresses. The edge-case of dealing with multiple `PaybuttonAddress`es is not being dealt with in this PR; I'll create another PR addressing it after solving #172  and #175.

**Test Plan:**
I will create another PR with the integration tests. For now:
1. Create a paybutton with a bitcash address, then click on it to go to its details.
https://user-images.githubusercontent.com/21281174/179738363-376d1efe-a76d-49d6-a9b0-c1846682c738.mp4
2. Send a request to sync this bch address transactions with for example
`curl -X GET http://localhost:3000/api/transactions/sync/bitcoincash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7q9ayphulp`
3. Check that the trasactions now appear in the button detail page.
![image](https://user-images.githubusercontent.com/21281174/179738845-b2af5f5e-a4eb-4f78-998b-53e537b92d38.png)
